docs/data_requirements.rst

```diff
@@ -0,0 +1,186 @@
+   enable them to conform to the requirements where they don't quite
+   satisfy them (for historical reasons or to abide by other/internal
+   data standards etc.). You can choose to enable these plugins via the
+   configuration. For more details, see the ``preprocess-mode-obs`` and
+   ``preprocess-mode-model`` configuration options as covered in the
+   :ref:`CLI <Command-line Interface (CLI) Reference>`.
+
+
+.. warning::
+
+   If you attempt to use the VISION Toolkit with model and/or observational
+   dataset which does not abide by the requirements documented here, it cannot
+   be guaranteed to work. (However, if you believe your dataset does in fact
+   meet these requirements and the toolkit is erroring or not seeming to
+   work correctly some way, please :ref:`contact us <IntroContact>`
+   so we can verify whether the dataset or the toolkit is
+   at issue).
+
+
+Checklist for validity of input datasets
+----------------------------------------
+
+Input datasets separately
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   These rules for validity apply whether the dataset in question is a model or
+   observational dataset, unless indicated otherwise.
+
+To work with the VISION toolkit, input datasets must satisfy the following.
+   
+1. Be CF Compliant, that is abiding by the
+   `CF Metadata Conventions <https://cfconventions.org/>`_.
+
+   * Ideally the dataset should abide by the latest version of the CF Conventions
+     (as per the document listed against 'latest released version' on the
+     `official website 'Conventions' page <https://cfconventions.org/conventions.html>`_),
+     but earlier versions are generally sufficient.
+
+2. Be in a form whereby, when aggregated during reading by ``cf.read`` or otherwise,
+   there is precisely one field per physical variable (e.g. one field only for
+   a temperature variable), which could be either from:
+
+   * The dataset already being in a form with one field per physical variable (preferred); OR
+
+   * The dataset being immediately aggregatable to that form (see for reference the
+     `cf-python aggregate function <https://ncas-cms.github.io/cf-python/function/cf.aggregate.html>`_
+     (``cf.aggregate``) which is run during the ``cf.read`` operation
+     which the VISION Toolkit uses to read-in datasets, and corresponding
+     `aggregation rules outlined here <https://ncas-cms.github.io/cf-python/aggregation_rules.html>`_); OR
+
+   * (Least desirable, but acceptable/supported) configuration being provided
+     to support a more relaxed aggregation of the dataset into that form, which
+     requires altering the aggregtion rule logic to be less strict to
+     reduce down the read-in fields more than the default aggregation rules
+     would, based upon provision of a dictionary of extra
+     keywords to provide to the `cf.aggregate` operation ran by the `cf.read`
+     used by the toolkit to read the dataset, see ``[[TODO NEW CONFIG INPUT]]``.
```

Comment:
@davidhassell we discussed supporting more relaxed aggregation back in mid 2024 when we thought in depth about this (according to my notes) but I haven't yet added in an option to enable extra keywords for the aggregate (either set so it happens in the initial `cf.read` if possible, else as a further `cf.aggregate` call after the read aiming to reduce the fields down further).

Do you think it is worth (it at this stage)? It wouldn't take long to implement, but equally we could save ourselves the work and extra complication by simply insisting the data aggregates down according to the default aggregation rules to the right form, i.e. satisfies either of the first two bullet points here and not allow the third option.